conf/messages.en

```diff
@@ -1705,7 +1713,6 @@ fileUpload.addressLine1.generic.error = address line 1 is missing or in the wron
 fileUpload.addressLine2.generic.error = address line 2 is missing or in the wrong format
 fileUpload.addressLine3.generic.error = address line 3 is missing or in the wrong format
 fileUpload.addressLine4.generic.error = address line 4 is missing or in the wrong format
-fileUpload.postCode.generic.error = postcode is missing or in the wrong format
```

Comment:
Why was this message key removed?

Reply:
It's a duplicate of the message on line 1697